man/scale_fill_gradient.Rd

```diff
@@ -2,7 +2,7 @@
 % Please edit documentation in R/scales.R
 \name{scale_fill_gradient}
 \alias{scale_fill_gradient}
-\title{redefine scale_fill_gradient}
+\title{redefine \code{ggplot2::scale_fill_gradient}}
```

Comment:
```suggestion
\title{redefine \code{ggplot2::scale_fill_gradient()}}
```

Reply:
fixed in [8eda9d7](https://github.com/inbo/INBOtheme/pull/23/commits/8eda9d78996d7cf4427e783d09944464d18f8839)